### PR TITLE
Fix Node Link on Container Detail View

### PR DESCRIPTION
**Pull request by alecmerdler**
### Description

Missing `/k8s/cluster` prefix.

Fixes https://jira.coreos.com/browse/CONSOLE-501

### CODE REVIEW

File: frontend/public/components/container.jsx [hunk 1]
```diff
@@ -166,7 +166,7 @@ const Details = (props) => {
         <h1 className="co-section-title">Network</h1>
         <dl className="co-m-pane__details">
           <dt>Node</dt>
-          <dd><Link to={`/nodes/${pod.spec.nodeName}`}>{pod.spec.nodeName}</Link></dd>
+          <dd><Link to={`/k8s/cluster/nodes/${pod.spec.nodeName}`}>{pod.spec.nodeName}</Link></dd>
```
Comment (spadgett) [inline frontend/public/components/container.jsx]: Any reason not to use `ResourceLink`?

Comment (alecmerdler) [reply frontend/public/components/container.jsx]: It wasn't being used before, so not sure. I just fixed the URL.

Comment (spadgett) [reply frontend/public/components/container.jsx]: I guess my point is that using `ResourceLink` would avoid bugs like this and we should switch (unless there's a reason it doesn't work here). Or possibly use use `resourcePath` to create the path.

Comment (benjaminapetersen) [reply frontend/public/components/container.jsx]: We tended to use helper functions in the old console to keep urls consistent & avoid mistakes, not sure if that would be useful here? Its easy to miss this stuff.

Comment (alecmerdler) [reply frontend/public/components/container.jsx]: Updated to use `<ResourceLink>`.

**Pull request merged**